### PR TITLE
Improve Streamlit landing page

### LINKED ISSUE #60: Implement and Improve Landing Page
**Context and feature:**
There is no true need to see the instructions of how to run the program if the program is already running. It is repetitive and it does not add anything to the landing page that is needed. 

**Proposed solution:**
We want to create a new landing page to reduce redundancy generated by the README. The new landing page would be a new markdown file and it would display how to use the sample files as an example of how to use files with the program.  It also would show quick instructions further explaining how to use the tools provided by GatorMiner.

### PR DESCRIPTION
**Pull request by Nathandloria**
In this pull request we created a new landing page to reduce redundancy generated by the README. The new landing page is in markdown format and it displays a lot of helpful information about the tool. It also shows quick instructions further explaining how to use the tools provided by GatorMiner.

## What is the current behavior?

closes #60 

## What is the new behavior if this PR is merged?

New, less redundant landing page.

Please describe the pull request as one of the following:

- [ ] Bug fix
- [ ] Breaking change
- [x] New feature
- [x] Documentation update
- [ ] Other 

### This PR has:

- [x] Commit messages that are correctly formatted
- [ ] Tests for newly introduced code
- [x] Docstrings for newly introduced code

#### Developers

@PaigeCD @caldwella2 @angusn8 @connellyw @Nathandloria

### CODE REVIEW

File: LANDING_PAGE.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+When `TF-IDF` is selected, the application will display a frequency matrix showing the correlation between documents. It does this buy dividing the frequency of the word by the total number of terms in a document.
+
+When `Spacy` is selected, the application will display a drop down named 'Model name' with two options:
+
+- `en_core_web_sm` which is used to produce a correlation matrix for **SMALLER** files. (10mb)
+- `en_core_web_md` which is used to produce a correlation matrix for **LARGER** files. (>10mb)
+
+**Warning exceeding these file limits could cause the program to crash.**
+
+<img src="resources/images/similarity.png" alt="browser" style="width:100%"/>
```
Comment (enpuyou) [inline LANDING_PAGE.md:116]: I didn't know about this before! It's a good thing to know. I would just like to maybe have a source that can refer to here.

**Review by enpuyou — CHANGES_REQUESTED** [review]
I haven't really gotten the chance to read into the details. Most are great, just one comment below. Another thing, can you create a folder named `docs`, and just put this markdown file in there. It seems like there will be more documentation coming up soon, so I would like to have them all in one place. Thanks!

**Comment by noorbuchi** [comment]
I think that your PR has multiple issues resulting from this commit 7b745607ca22d3f3d2b33da6afe547bff2673708 . This reverts very important fixes. Please make sure to resolve this by restoring the deleted files and re-adding the important fixes

**Review by noorbuchi — CHANGES_REQUESTED** [review]
I think the new landing page looks great! I requested few small changes that you should address before merging this PR. Thanks for working on this helpful documentation!

**Review by enpuyou — APPROVED** [review]
Hi @Nathandloria, I think this PR is good! I have just added an `mdl` check into the Travis CI, so it would enforce markdown rules on documents in `docs` folder. Should anything fail (there is probably not, but just in case), just do a quick fix, I don't think it would be anything substantial. Thanks again!

**Review by corlettim — APPROVED** [review]

**Review by munzekm — APPROVED** [review]

**Review by noorbuchi — APPROVED** [review]

**Pull request merged**